### ISSUE TITLE
MockModule  no Directive annotation found

### ISSUE THREAD

**Issue by kekel87** [ike18t/ng-mocks#19]
Hello,
I try to mock a entire module using `MockModule(MyModule)` function.

But I get : 
```
Failed: No Directive annotation found on [NgClass, NgComponentOutlet, NgForOf, NgIf, NgTemplateOutlet, NgStyle, NgSwitch, NgSwitchCase, NgSwitchDefault, NgPlural, NgPluralCase]
```

I think CommonModule is missing somewhere. This may be a link to issue #1.
Thank you for your help.

**Comment by ike18t** [comment]
I went ahead and updated MockModule to never mock CommonModule.

Fixed with release of 6.2.2
cea5055b8194c12a7ca6ecd31182d9454941c8ed

**Comment by rwpino** [comment]
I m having this issue with MockDirective method

```
Uncaught Error: No Directive annotation found on e
    at t.e.resolve (main.ef271016ceb35fa51ea5.js:formatted:18905)
    at t.resolve (main.ef271016ceb35fa51ea5.js:formatted:19267)
    at Object.c [as MockDirective] (main.ef271016ceb35fa51ea5.js:formatted:158611)
    at Object.RnxJ (main.ef271016ceb35fa51ea5.js:formatted:103344)
    at f (runtime.bf0ad6a492d455f9de3a.js:1)
    at Object.GmxJ (main.ef271016ceb35fa51ea5.js:formatted:60193)
    at f (runtime.bf0ad6a492d455f9de3a.js:1)
    at Object.Ss9G (main.ef271016ceb35fa51ea5.js:formatted:104314)
    at f (runtime.bf0ad6a492d455f9de3a.js:1)
    at Object.zUnb (main.ef271016ceb35fa51ea5.js:formatted:191690)
```

this issue is coming when I compile for production
